solutions/_snippets/search-cases.md

```diff
@@ -0,0 +1,3 @@
+Use the search bar on the **Cases** page to quickly find cases and case data. You can search for case titles, descriptions, and IDs. Keywords and text search is supported.
```

Comment:
@christineweng is the search case-sensitive? From my testing, it doesn't look to be, but I still wanted to double-check with you.

Reply:
Hey @nastasha-solomon , good question, some of them were (title, description, comments), and some were keyword search and had to be exact (case ids, alert ids, event ids, observables, custom fields)

I opened https://github.com/elastic/kibana/pull/246643 to update observables and custom fields search to be case insensitive. As discussed in slack, ids are shown as lower case throughout kibana, user is very unlikely to get a upper case id, hence keeping them as is.

Reply:
@christineweng how's this:

```suggestion

The Cases page has a search bar for quickly finding cases and case data. You can search for case titles, descriptions, and IDs using keywords and text. Note the following rules for search:

* **Keywords**: Searches for keywords (like case and alert IDs) must be exact.
* **Text**: Text searches (such as case titles and descriptions) are case-insensitive.
* **Syntax**: No special syntax is required when entering your search criteria.
```